### ISSUE TITLE
Fails to build with libxc >5.1.3

### ISSUE THREAD

**Issue by awvwgk** [qcserenity/serenity#7]
Some functional symbols have been renamed from 5.1.4 onward, which leads to a build failure in serenity due to missing macros:

- ``XC_LDA_XC_BN05``
- ``XC_GGA_X_LC2GAU``
- ``XC_GGA_X_LCGAU``
- ``XC_GGA_X_LCGAU_CORE``
- ``XC_GGA_XC_LB07``

A possible fix could be

````c
#ifdef XC_GGA_XC_LB07
alias = XC_GGA_XC_LB07;
#else
alias = XC_HYB_GGA_XC_LB07;
#endif

**Comment by nabbelbabbel** [comment]
Thanks for the hint.
The intention was not to support ranges of libxc but, instead, to fix the version.
There may be an update of the requested version in Serenity at some point (possibly the release after the next one).
I will keep this in mind.

- Jan

**Comment by nabbelbabbel** [comment]
This issue has been addressed in Serenity v1.5.0 .
Thanks for the feedback.
